Library/Formula/dnscrypt-proxy.rb

```diff
@@ -39,6 +60,10 @@ def install
 
     system "./configure", *args
     system "make", "install"
+
+    (libexec/"update-resolvers.sh").write updateresolvers
+    chmod 0774, libexec/"update-resolvers.sh"
+    bin.write_exec_script libexec/"update-resolvers.sh"
```

Comment:
I'd probably drop the `.sh` here so people don't have to remember that part. Plenty of shell-only executables that aren't called without it.